.travis.yml

```diff
@@ -7,26 +7,25 @@ notifications:
 node_js:
 - 8.5.0
 cache:
+  yarn: true
   directories:
   - node_modules
-before_script:
-- npm install -g gulp
 before_install:
-- npm install -g npm@5.5.1
+  - npm install -g npm@5.5.1
```

Comment:
Do we still need to instal npm then ? :)

Reply:
Well I kept that line in case we needed a specific version for the deployment.Do you know why we were installing this specific version?

Reply:
Ahh I see.

I don't think there is a particular reason except that it's the latest "official". We can leave it there I guess.